src/FamilyHubs.ReferralUi.Ui/Pages/Error/404.cshtml

```diff
@@ -0,0 +1,24 @@
+﻿@page
+@{
+    ViewData["Title"] = "Page not found";
+}
+
+<div class="govuk-width-container">
```

Comment:
I've just updated FX to remove these two lines (otherwise you end up with 2 main elements in the page and incorrect white-space).

Reply:
Same with the error pages.

Reply:
See https://github.com/DFE-Digital/fh-service-directory-ui/pull/30/files#diff-da91d8ff8e2c20f5273779b463c6af08ac25baf23d7549dc01425d8da322a922

Reply:
@lazcool fix this